module/applications/actor/api/base-actor-sheet.mjs

```diff
@@ -196,6 +211,7 @@ export default class BaseActorSheet extends PrimarySheetMixin(
     // Cache concentration data and prepare items
     this._concentration = this.actor.concentration;
     await this._prepareItems(context);
+    context.spellbook = this._prepareSpellbook(context);
```

Comment:
Should probably leave this to the subclassed sheets to call so we don't end up preparing the spellbook for actor types that don't have it like vehicles and groups..